configure

```diff
@@ -174,6 +174,19 @@ while true; do
 
   if [[ -z "$TF_CUDNN_VERSION" ]]; then
     TF_CUDNN_EXT=""
+    # Resolve to the SONAME of the symlink.  Use readlink without -f
+    # to resolve exactly once to the SONAME.  E.g, libcudnn.so ->
+    # libcudnn.so.4
+    REALVAL=`readlink ${CUDNN_INSTALL_PATH}/lib64/libcudnn.so`
```

Comment:
Ah, if there's no symlink by default, then this branch won't execute, so we're okay.  If the symlink does exist, the existing codepath will work too.